internal/plugins/plugin_spec_test.go

```diff
@@ -26,6 +26,34 @@ func TestValidate(t *testing.T) {
 	}
 }
 
+func TestValidateDefaultValues(t *testing.T) {
+	file := getTestSpecFileWithoutDefaults(t)
+	testPluginSpec, err := unmarshal(file)
+	if err != nil {
+		t.Fatal("failed to unmarshal test file")
+	}
+	var tests = []PluginSpec{
+		PluginSpec{},
+		*testPluginSpec,
+	}
+	for i, test := range tests {
+		t.Run(fmt.Sprintf("test%d", i), func(t *testing.T) {
+			err = test.validate()
+			if err != nil {
+				t.Fatalf("expected nil but got %v", err)
+			}
+		})
+	}
+}
+
+func getTestSpecFileWithoutDefaults(t *testing.T) []byte {
```

Comment:
We might be able to extend the `getTestSpecFile` function with a filename option and the filenames can be named constants to get the same outcome. This way we can add more test files later and reuse the existing method.